### PR TITLE
Css refactor

### PR DESCRIPTION
**Pull request by vineet-pruthi**

### CODE REVIEW

File: otcextensions/osclient/css/v1/snapshot.py [hunk 1]
```diff
@@ -269,17 +240,19 @@ def get_parser(self, prog_name):
...
         )
         parser.add_argument(
             '--keep-days',
             metavar='<keep_days>',
             type=int,
-            required=True,
-            help=('Number of days that a snapshot can be retained.')
+            default=15,
+            help=('Number of days that a snapshot can be retained. '
+                  '(Default Value: 15).')
```
Comment (anton-sidelnikov) [inline otcextensions/osclient/css/v1/snapshot.py]: why you defaulted this parameters? 15 for keep_days it is a big period.

File: otcextensions/sdk/css/v1/cluster.py [hunk 2]
```diff
@@ -74,142 +53,110 @@ class Cluster(resource.Resource):
...
+    cmk_id = resource.Body('cmk_id')
     #: Cluster creation time
     created_at = resource.Body('created')
     #: Type of the data search engine
-    datastore = resource.Body('datastore', type=DatastoreSpec)
-    #: Disk encryption specification
-    disk_encryption = resource.Body('diskEncryption', type=DiskEncryption)
+    datastore = resource.Body('datastore', type=dict)
+    #: Disk Encryption Object. (For request body only)
+    diskEncryption = resource.Body('diskEncryption', type=dict)
```
Comment (anton-sidelnikov) [inline otcextensions/sdk/css/v1/cluster.py]: disk_encryption python style

File: otcextensions/sdk/css/v1/cluster.py [hunk 3]
```diff
@@ -74,142 +53,110 @@ class Cluster(resource.Resource):
...
-    nodes = resource.Body('instances', type=list, list_type=dict)
-    #: Number of cluster instances (1..32)
-    instance_count = resource.Body('instanceNum', type=int)
-    #: Disk encryption flag (read only)
+    #: Public IP address.
+    floating_ip = resource.Body('publicIp')
+    #: Cluster Node (request Body)
+    instance = resource.Body('instance', type=dict)
+    #: Number of cluster Nodes. The value range is 1 to 32. (request Body)
+    instanceNum = resource.Body('instanceNum', type=int)
```
Comment (anton-sidelnikov) [inline otcextensions/sdk/css/v1/cluster.py]: instance_num

File: otcextensions/tests/unit/osclient/css/v1/test_cluster.py [hunk 4]
```diff
@@ -22,54 +23,56 @@
...
-        'Type',
-        'Version',
+        'DataStore',
         'Status',
-        'Created At'
+        'Created At',
     )
 
-    columns = ('id', 'name', 'type', 'version', 'status', 'created_at')
+    # columns = ('id', 'name', 'datastore', 'status', 'created_at')
```
Comment (anton-sidelnikov) [inline otcextensions/tests/unit/osclient/css/v1/test_cluster.py]: ?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: otcextensions/osclient/css/v1/snapshot.py (hunk 1 above)
Comment (vineet-pruthi) [reply otcextensions/osclient/css/v1/snapshot.py]: Removed default value for --keep-days

File: otcextensions/sdk/css/v1/cluster.py (hunk 3 above)
Comment (vineet-pruthi) [reply otcextensions/sdk/css/v1/cluster.py]: Updated to instance_num.

File: otcextensions/sdk/css/v1/cluster.py (hunk 2 above)
Comment (vineet-pruthi) [reply otcextensions/sdk/css/v1/cluster.py]: Updated.

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**